server/db/db.js

```diff
@@ -84,10 +97,53 @@ async function execute(path, params = {}) {
   sql = sql.replace(/\$\{[^{}]+\}/g, queryParam);
   const values = queryVariables ? queryVariables.map(p => params[p]) : [];
   return query(sql, values);
-}
+};
+
+const migrate = async () => {
+  let existingMigrations = [];
+  try {
+    const result = await pool.query('SELECT * FROM migrations');
+    existingMigrations = result.rows.map((r) => r.file_name);
+  } catch {
+    console.log('First migration');
+  }
+
+  // Get outstanding migrations
+  const outstandingMigrations = await migrationUtils.getOutStandingMigrations(existingMigrations);
```

Comment:
see above suggestion

```suggestion
  const outstandingMigrations = await getOutStandingMigrations(existingMigrations);
```